docs/requirements.txt

```diff
@@ -3,3 +3,5 @@ sphinx-rtd-theme==1.0.0
 # dependencies from setup.py
 py-algorand-sdk>=1.9.0,<2.0.0
 semantic-version>=2.9.0,<3.0.0
+# dependencies from ./requirement.txt
+docstring-parser==0.14.1
```

Comment:
ref: https://stackoverflow.com/questions/15889621/sphinx-how-to-exclude-imports-in-automodule

This line is simply put to make all package imported for pyteal source code, so build doc could test pass